docs/resources/environment.md

```diff
@@ -64,7 +64,7 @@ resource "env0_environment" "example_with_hcl_configuration" {
 - **auto_deploy_on_path_changes_only** (Boolean) redeploy only on path changes only
 - **configuration** (Block List) terraform and environment variables for the environment (see [below for nested schema](#nestedblock--configuration))
 - **deploy_on_push** (Boolean) should run terraform deploy on push events
-- **force_destroy** (Boolean) destroy safegurad
+- **force_destroy** (Boolean) Destroy safeguard. Must be enabled before delete/destroy
```

Comment:
@liranfarage89 the markdown files are auto-generated so we should not update them manually

Reply:
[f77799a](https://github.com/env0/terraform-provider-env0/pull/421/commits/f77799ac489cd2716be2f7a52f81b3500b5e6a06)
@yaronya